src/effects/memcached.js

```diff
@@ -0,0 +1,13 @@
+import Memcached from 'memcached-promisify'
+
+const memcached = ({
+  memcachedHost,
+  memcachedHosts,
+  memcachedOptions
+}) => {
+  const host = memcachedHost || memcachedHosts
```

Comment:
I'm a little confused by this. How come the singular host is more important? Mainly wondering why the distinction

Reply:
I just want it to be generally useful. It's kinda an interpretation of the underlying API at https://github.com/3rd-Eden/memcached I guess.

Reply:
For them it's an un-named option as an argument to an fn. For us it gets named in a config file so I want it to be useful to us in our single-host universe but also be able to take advantage of the underlying lib.